src/app/campaign-info/campaign-info.component.ts

```diff
@@ -42,8 +52,26 @@ export class CampaignInfoComponent implements OnInit {
     this.campaign = this.route.snapshot.data.campaign;
     this.campaignOpen = CampaignService.isOpenForDonations(this.campaign);
     this.campaignFinished = CampaignService.isInPast(this.campaign);
-    this.campaignTarget = this.currencyPipe.transform(this.campaign.target, this.campaign.currencyCode, 'symbol', '1.0-0') as string;
-    this.campaignRaised = this.currencyPipe.transform(this.campaign.amountRaised, this.campaign.currencyCode, 'symbol', '1.0-0') as string;
+    this.campaignTarget = this.currencyPipe.transform(
+      this.campaign.parentUsesSharedFunds ? this.campaign.parentTarget : this.campaign.target,
+      this.campaign.currencyCode,
+      'symbol',
+      '1.0-0',
+    ) as string;
```

Comment:
do you know why we need the `as string` here? I'm not sure why currencyPipe,transform returns a nullable.

Reply:
I see the `as string` is in the old version of the code just as much though so this shouldn't be a blocker to the PR.

Reply:
Not entirely sure @bdsl, this is an existing bit I just reformatted. I think maybe pipes give null output from null input, but because of the resolver and snapshot we know that our input isn't null if we got this far?

Reply:
Right - I hadn't kept track of the fact that its not new.